src/main/java/com/rultor/agents/github/qtn/QnMerge.java

```diff
@@ -77,25 +76,34 @@ public Req understand(final Comment.Smart comment,
                 this, "merge request found in %s#%d, comment #%d",
                 issue.repo().coordinates(), issue.number(), comment.number()
             );
-            if (QnMerge.allChecksSuccessful(issue.pull())) {
+            final CheckablePull pull = new CheckablePull(issue.pull());
+            if (pull.containsFile(".rultor.yml")) {
                 new Answer(comment).post(
-                    true,
-                    String.format(
-                        QnMerge.PHRASES.getString("QnMerge.start"),
-                        home.toASCIIString()
-                    )
-                );
-                req = QnMerge.pack(
-                    comment,
-                    issue.repo().pulls().get(issue.number())
-                );
-            } else {
-                new Answer(comment).post(
-                    false,
-                    QnMerge.PHRASES.getString("QnMerge.checks-are-failed")
+                        false,
+                        QnMerge.PHRASES
+                                .getString("QnMerge.system-files-affected")
                 );
                 req = Req.DONE;
-            }
+            } else
+                if (pull.allChecksSuccessful()) {
+                    new Answer(comment).post(
+                            true,
```

Comment:
@pnatashap indentation here is definitely wrong (not 4 spaces), but I can't understand why Qulice is not complaining about this...